### PR TITLE
Fix links and auto update spacing

### PR DESCRIPTION
**Pull request by holdenk**
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/pipelines/2870)

### CODE REVIEW

File: backend/api/job.proto [hunk 1]
```diff
@@ -32,9 +32,9 @@ option (grpc.gateway.protoc_gen_swagger.options.openapiv2_swagger) = {
     key: "default";
     value: {
       schema: {
-        json_schema: {
-          ref: ".api.Status";
-        }
+	json_schema: {
```
Comment (numerology) [inline backend/api/job.proto]: Can you justify why this is the preferred indent?

Comment (rmgogogo) [reply backend/api/job.proto]: I may prefer the existing indent. Can't understand why send out this PR.
Is it due to PR author's IDE/editor setting?

File: backend/api/job.proto [hunk 2]
```diff
@@ -167,7 +167,7 @@ message CronSchedule {
   google.protobuf.Timestamp end_time = 2;
 
   // The cron string. For details how to compose a cron, visit
-  // ttps://en.wikipedia.org/wiki/Cron
+  // https://en.wikipedia.org/wiki/Cron
```
Comment (rmgogogo) [inline backend/api/job.proto]: +1 to this fix. thanks!

**Comment by rmgogogo** [comment]
kindly ping
/retest

**Pull request closed without merging**